components/service/pocket/src/test/java/mozilla/components/service/pocket/update/RefreshSpocsWorkerTest.kt

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.pocket.update
+
+import mozilla.components.service.pocket.helpers.assertClassVisibility
+import org.junit.Test
+import kotlin.reflect.KVisibility.INTERNAL
+
+class RefreshSpocsWorkerTest {
+    @Test
+    fun `GIVEN a RefreshSpocsWorker THEN its visibility is internal`() {
+        assertClassVisibility(RefreshSpocsWorker::class, INTERNAL)
+    }
```

Comment:
Could we extend the test cases to also covered that we are returning different results? Some examples can be [found](https://github.com/mozilla-mobile/android-components/blob/main/components/feature/addons/src/test/java/mozilla/components/feature/addons/migration/SupportedAddonsWorkerTest.kt#L63). :)

Reply:
No problem!